src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -1000,6 +1117,12 @@ inline void handleKW( SummaryConfig::keyword_list& list,
 // =====================================================================
 
 SummaryConfigNode::Type parseKeywordType(std::string keyword) {
+    if (is_well_completion(keyword))
+        keyword = keyword.substr(0, keyword.size() - 1);
+
+    if (is_connection_completion(keyword))
+        keyword = keyword.substr(0, keyword.size() - 1);
```

Comment:
You might consider using `std::string::pop_back()` instead of these `.substr()` calls:
```C++
if (is_well_completion(keyword) || is_connection_completion(keyword))
    keyword.pop_back();  // Remove trailing 'L' character
```

Reply:
Good idea - will do it.